assignments/objects.js

```diff
@@ -18,16 +18,26 @@ const example = {
 }
 
 // Write your intern objects here:
```

Comment:
Good start here, you should look into class constructor and prototypes to save on some typing and make your code dry.